Open hackathon registration (#2264)
nf-core/website: src/content/events/2024/hackathon-march-2024/index.mdx

endDate: '2024-03-20'
endTime: '17:00+02:00'
locationName: Gather town, Slack, and local sites.
importTypeform: true
announcement:

>     text: '✏️ Register your local site for the March hackathon!'

Reviewer: ```suggestion
    text: 'Sign up for the March hackathon is open!'
```

endDate: '2024-03-20'
endTime: '17:00+02:00'
locationName: Gather town, Slack, and local sites.
importTypeform: true
announcement:

>     text: 'Sign-up for the March hackathon is open!'

Reviewer 2: ```suggestion
    text: 'Registration for the online / distributed March hackathon is now open!'
```

The nf-core hackathon is free and is being held online (we will be back in [Gather](https://gather.town/)).
Some community members are also hosting local sites.

**Key Dates**

> -   Registration to host a local site opens (**Jan 15, 2024**)
> -   Registration to host a local site closes (**Feb 2, 2024**)

Reviewer 2: May as well indicate that these have passed..

```suggestion
-   ~~Registration to host a local site opens (**Jan 15, 2024**)~~
-   ~~Registration to host a local site closes (**Feb 2, 2024**)~~
```
